src/components/pacbio/PacbioPoolAliquotEdit.vue

```diff
@@ -40,8 +40,19 @@
         :with-icon="isValidationExists('volume')"
       >
         <traction-input v-model="volume" data-attribute="volume" placeholder="Volume" />
+        <div class="flex items-center px-1">
+          <traction-tooltip
+            v-show="aliquot.available_volume"
```

Comment:
Think this should be something like `aliquot.available_volume != null` otherwise it wont show when available_volume is 0